.github/workflows/release-ticket-creation.yml

```diff
@@ -3,33 +3,36 @@ on:
   workflow_dispatch:
     inputs:
       branch_name:
-        description: 'Release target branch name'
+        description: 'Type a branch name starting with `release/v`'
```

Comment:
This field is displayed in the workflow trigger popup UI like this. 
So I aded a short guide message to make the us type `release/v` branch name. (Too bad that I can set `placeholder` 🫠

<img width="943" alt="Screenshot 2023-06-27 at 6 02 48 PM" src="https://github.com/sendbird/sendbird-uikit-react/assets/10060731/908700bf-6091-4707-8a7a-e064b0e095f9">